warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__demand_responsive_routes_match.sql

```diff
@@ -1,7 +1,7 @@
 WITH
 
 idx AS (
-    SELECT * FROM {{ ref('int_gtfs_quality__gtfs_service_data_guideline_index') }}
+    SELECT * FROM {{ ref('int_gtfs_quality__gtfs_service_data_schedule_guideline_index') }}
```

Comment:
Does this need to be created in this PR? Sorry if I am missing something, I don't see an existing table with this name (not sure if there's just a commit that needs to be pushed?)

Reply:
yes it sure does, added it to the PR now, thanks!